packages/fxa-settings/src/components/Settings/FlowRecoveryKeyHint/index.tsx

```diff
@@ -194,6 +198,7 @@ export const FlowRecoveryKeyHint = ({
             <button
               className="cta-primary cta-xl w-full mt-6 mb-4"
               type="submit"
+              disabled={account.loading}
```

Comment:
Do we need this check now?

Reply:
Thanks for catching! This should be the `isLoading` state.